Source/Core/DolphinQt/Achievements/AchievementLeaderboardWidget.cpp

```diff
@@ -50,7 +51,8 @@ void AchievementLeaderboardWidget::UpdateData(bool clean_all)
     for (u32 bucket = 0; bucket < leaderboard_list->num_buckets; bucket++)
     {
       const auto& leaderboard_bucket = leaderboard_list->buckets[bucket];
-      m_common_layout->addWidget(new QLabel(tr(leaderboard_bucket.label)));
+      m_common_layout->addWidget(new QLabel(tr(leaderboard_bucket.label)), row, 0);
+      row += 2;
```

Comment:
Was this added on purpose?  Seems odd, as your below `if (row > 0)` check will always be true.  Also, what about row 0 and 1 in the RA content?  This id eventually trickles its way down to a rc call I believe.

Reply:
I can drop the check yes, but this was deliberate - I want the bucket header to show up in its own row (row 0), and then the dialog will draw a divider line (row 1) before going into the first leaderboard (row 2). I'll double check that I'm not accidentally off by one on the leaderboards though.

Reply:
Verified, none of the leaderboards are missing. Did remove the checks because row will always be > 0 at those points. But row is set independent of leaderboard order (and saved in a map for future reference) so everything seems complete here.

Reply:
Alright, seems odd that somehow RA accounts for the view?  But if you've confirmed it's all there, then I guess it's good.

Reply:
Not sure what you mean by "the view"? You say this trickles down to an RC call; nothing in this popup calls back to RC I believe.

Reply:
Hmm, yeah, looking again seems like I'm mistaken.  Sorry.  By "view" I meant the view in MVVM paradigm.  The view is usually just the UI, which has no relation to the data.  Good UI design usually dictates a separation between the two.  Anyway, apologies, this should be good to go.